integration/container/health_test.go

```diff
@@ -32,6 +33,34 @@ func TestHealthCheckWorkdir(t *testing.T) {
 	poll.WaitOn(t, pollForHealthStatus(ctx, client, cID, types.Healthy), poll.WithDelay(100*time.Millisecond))
 }
 
+// GitHub #37263
+// Do not stop healthchecks just because we sent a signal to the container
+func TestHealthKillContainer(t *testing.T) {
+	defer setupTest(t)()
+
+	ctx := context.Background()
+	client := testEnv.APIClient()
+
+	id := container.Run(ctx, t, client, func(c *container.TestContainerConfig) {
```

Comment:
Can you push a separate commit to fix the signature for 18.09?

```suggestion
	id := container.Run(t, ctx, client, func(c *container.TestContainerConfig) {
```

Similar to https://github.com/docker/engine/pull/286/commits/584c0857ab21895e62feac686448085113c6c977

(Keeping the fixup commit separate would help reverting it if those signature-changes are backported to 18.09)